extensions/traceability.sigmf-ext.md

```diff
@@ -0,0 +1,200 @@
+# The `traceability` SigMF Extension Namespace v1.0.0
+
+This document proposes a new extension namespace called `traceability` for the Signal Metadata
+Format (SigMF) specification. This extension provides traceability information for the metadata.
+
+## 1 Global
+
+`traceability` extends the [Global](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#global-object) object.
+
+The following fields are added to the `global` object:
+
+|name|required|type|description|
+|----|--------|----|-----------|
+|`traceability:last_modified`|false|[DataChange](#datachange-object)|Captures the author and timestamp of the most recent modification|
+|`traceability:last_reviewed`|false|[DataChange](#datachange-object)|Captures the author and timestamp of the most recent review|
+|`traceability:revision`|false|integer|Specifies the revision number associated with the metadata|
+|`traceability:origin`|false|[Origin](#origin-object)|Provides information about the origin of the data|
+|`traceability:sample_length`|false|integer|Specifies the total number of samples of the Dataset|
```

Comment:
I don't think this field belongs here. SigMF has very deliberately avoided direct references to this, and its not clear how knowing this value helps with this extensions objectives.